test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2507,7 +2507,7 @@ static public void testCheckCastValueTypeOnInvalidQtype() throws Throwable {
 	/*
 	 * Ensure that casting null to a value type class will throw a null pointer exception
 	 */
-	@Test(priority=1, expectedExceptions=NullPointerException.class)
+	@Test(enabled=false, priority=1, expectedExceptions=NullPointerException.class)
```

Comment:
Please add a comment indicating why this test has to be disabled.

Reply:
Addressed in 156b344